test/tests/serial/mail_with_imap.bats

```diff
@@ -38,7 +38,7 @@ teardown_file() {
 }
 
 @test "checking saslauthd: rimap smtp authentication" {
-  run docker exec mail_with_imap /bin/sh -c "nc -w 5 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/smtp-auth-login.txt | grep 'Authentication successful'"
+  run docker exec mail_with_imap /bin/sh -c "nc -w 5 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/smtp-auth-login.txt | grep 'Error: authentication not enabled'"
```

Comment:
If you want to change tests to fail on port 25 when they try to authenticate, then you should also include a test that proves the feature can authenticate successfully on a different port?

Reply:
Yes, I'm agree with it. Added